src/components/accession2/properties/CollectionSource.tsx

```diff
@@ -1,6 +1,11 @@
-import React from 'react';
+import React, { useState } from 'react';
 import strings from 'src/strings';
 import { Dropdown } from '@terraware/web-components';
+import TooltipLearnMoreModal, {
+  LearnMoreLink,
+  LearnMoreModalContentCollectionSource,
+  TooltipLearnMoreModalData,
+} from '../../TooltipLearnMoreModal';
```

Comment:
another way (if you want to avoid relative paths) `from '/src/components/TooltipLearnMoreModal`